80s Game/Assets/Scripts/GameManager.cs

```diff
@@ -239,4 +239,12 @@ public void StartRoundDelay()
         float delay = roundEndTheme.Clip.length - (float)roundEndTheme.EndOffset;
         StartCoroutine(ActiveGameMode.DelayNextRound(delay));
     }
+
+    /// <summary>
+    /// General call for debug lines for non-monobehavior scripts
+    /// </summary>
+    public void AddDebugMessage(string msg)
```

Comment:
That's really smart. I like this.

Reply:
Thanks, I was sick of not being able to run debug messages in our other scripts lol. Was very helpful